portfolio/bin/src/main/webapp/projects.html

```diff
@@ -0,0 +1,32 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Lilian Hunt</title>
```

Comment:
Please fix the 4-space problem throughout this file.

Reply:
fixed